src/lib/components/Header.svelte

```diff
@@ -1,15 +1,19 @@
 <script lang="ts">
+	import { goto } from '$app/navigation';
 	import { page } from '$app/stores';
 	import * as modal from '$lib/utils/modal';
-	import { goto } from '$app/navigation';
-	import Connect from '$lib/shared/Connect.svelte';
-	import Create from '@components/Create/index.svelte';
+	import Protected from '$components/Protected.svelte';
+	import Connect from '$components/Connect.svelte';
+	import Create from '$components/Create/index.svelte';
 </script>
 
 <header>
 	<div class="home" class:active={$page.url.pathname === '/'}>
 		<a sveltekit:prefetch href="/"><h3>👔 Workstreams</h3></a>
-		<button on:click={() => modal.show(Create)}>Create</button>
+		<Protected>
+			<button on:click={() => modal.show(Create)}>Create</button>
+			<p slot="unauthorized">Please connect your wallet.</p>
```

Comment:
```suggestion

```

We don't need this. We'll just not show the button when you're not connected (or we show a connect modal when you press it)